components/esp_websocket_client/test/host_test/main/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+idf_component_register(SRCS "main.c"
+                    INCLUDE_DIRS
+                    "."
+                    REQUIRES esp_websocket_client)
+target_compile_options(${COMPONENT_LIB} PRIVATE "-Wno-format")
+target_compile_definitions(${COMPONENT_LIB} PRIVATE "-DCONFIG_IDF_TARGET_LINUX")
```

Comment:
This shouldn't be needed. The build system should add this (might need to include `sdkconfig.h`)